Lib/pdb.py

```diff
@@ -138,7 +138,13 @@ def find_function(funcname, filename):
 
             if funcdef:
                 try:
-                    funccode = compile(funcdef, filename, 'exec').co_consts[0]
+                    code = compile(funcdef, filename, 'exec')
+                    for const in code.co_consts:
+                        if isinstance(const, CodeType) and const.co_name == funcname:
+                            funccode = const
+                            break
+                    else:
+                        continue
```

Comment:
I think this is easier to read:

```suggestion
                    try:
                        next(c for c in code.co_consts if isinstance(c, CodeType) and c.co_name == funcname)
                    except StopIteration:
                        continue
```

Reply:
I don't that's equivalent. The assignment to `funccode` is missing. Personally I think the original is easier to read. The modified one is definitely shorter, but the usage of `next` and `StopIteration` is pretty tricky.

Reply:
Hmm, I don't think the `continue` part is correct, that should not happen. I'll try to fix it.